designer/server/src/main/scala/pl/touk/nussknacker/ui/definition/ScenarioPropertiesConfigFinalizer.scala

```diff
@@ -3,16 +3,23 @@ package pl.touk.nussknacker.ui.definition
 import cats.implicits.catsSyntaxSemigroup
 import pl.touk.nussknacker.engine.api.component.{AdditionalUIConfigProvider, ScenarioPropertyConfig}
 import pl.touk.nussknacker.engine.api.process.ProcessingType
+import pl.touk.nussknacker.engine.api.properties.ScenarioProperties
 
 class ScenarioPropertiesConfigFinalizer(
     additionalUIConfigProvider: AdditionalUIConfigProvider,
     processingType: ProcessingType
 ) {
 
-  def finalizeScenarioProperties(
-      scenarioPropertiesConfig: Map[String, ScenarioPropertyConfig],
+  def finalizeScenarioPropertiesParameters(
```

Comment:
The previous name was better. Why we have properties parameters now and they are available only here? 
In our domain we have:
1. Parameters of nodes
2. Properties of scenarios / fragments

Let's not mix these things

Reply:
I did it to avoid mixing it with finalizing `ScenarioProperties` object. But I guess there's no good way out of this misnaming. I have reverted this.